client/src/app/site/agenda/services/agenda-repository.service.ts

```diff
@@ -18,15 +22,20 @@ import { CollectionStringModelMapperService } from '../../../core/services/colle
     providedIn: 'root'
 })
 export class AgendaRepositoryService extends BaseRepository<ViewItem, Item> {
-    public constructor(DS: DataStoreService, mapperService: CollectionStringModelMapperService) {
+    public constructor(
+        protected DS: DataStoreService,
+        private httpService: HttpService,
+        mapperService: CollectionStringModelMapperService
+    ) {
         super(DS, mapperService, Item);
     }
 
     /**
      * Returns the corresponding content object to a given {@link Item} as an {@link AgendaBaseModel}
      * @param agendaItem
+     * Used dynamically because of heavy race conditions
```

Comment:
proper `@returns` is missing in the documentation

Reply:
yep